prow/cmd/sub/main.go

```diff
@@ -81,7 +81,7 @@ func init() {
 	fs.StringVar(&flagOptions.configPath, "config-path", "/etc/config/config.yaml", "Path to config.yaml.")
 	fs.StringVar(&flagOptions.jobConfigPath, "job-config-path", "", "Path to prow job configs.")
 
-	fs.BoolVar(&flagOptions.dryRun, "dry-run", true, "Dry run for testing. Uses API tokens but does not mutate.")
+	fs.BoolVar(&flagOptions.dryRun, "dry-run", false, "Dry run for testing. Uses API tokens but does not mutate.")
```

Comment:
Technically should go to `ANNOUNCEMENTS.md`

Reply:
I think this is a bad, unsafe default. Especially as it differs from all the other instances of the flag, right?

Reply:
Just for the sake of argument https://github.com/kubernetes/test-infra/blob/master/prow/cmd/crier/main.go#L108. I find it weird to have a boolean flag being true by default since you're supposed to --boolflag to enable it. Plus in every cli I used --dry-run is false by default.

Reply:
Although I am all about consistency, so your wish is my command.

Reply:
This is why I always create `--confirm` flags in favor of `--dry-run=false`

Reply:
IMO the flag in crier also has an unsafe default... although just my luck I bet I lgtm'd the change  :)

Reply:
Reverted for consistency